src/AzureAuth/Program.cs

```diff
@@ -97,6 +97,12 @@ private static void Main(string[] args)
             services.AddSingleton<IPublicClientAuth, PublicClientAuth>();
             services.AddSingleton<ILogger>(logger);
 
+            Console.CancelKeyPress += (_, _) =>

```

Comment:
I tried testing this via your branch and behavior was same in main / when checking out your branch.

I tried running `azureauth ado token` and abruptly pressed CTRL+C (version 0.8.2) and no message was printed. What am I missing?

Reply:
Maybe you interrupt AzureAuth too quick, and at that time logger hasn't been initialized. You can add option `--verbosity debug`, and press Ctrl+C after you see the first message.